packages/app/jsonmodels/epochs.go

```diff
@@ -5,16 +5,20 @@ import (
 )
 
 type EpochInfo struct {
-	EI     uint64 `json:"Index"`
-	ECR    string `json:"RootsID"`
-	PrevEC string `json:"prevEC"`
+	ID               string `json:"id"`
+	Index            uint64 `json:"index"`
+	ECR              string `json:"rootsID"`
```

Comment:
Maybe just `roots` or `rootsID`? It should be consistent with the naming that we use inside the protocol, everything else is confusing